test/integration/features/addon-registry-route.feature

```diff
@@ -0,0 +1,45 @@
+@addon-registry-route @addon
+Feature: registry-route add-on
+registry-route exposes private docker registry,
+so that user can push their images to private registry and delpoy it.
+
+  Scenario: As a user, I can enable the registry-route add-on
+    Given user starts shell instance on host machine
+     When executing "minishift addons enable registry-route" succeeds
+     Then stdout should contain "Add-on 'registry-route' enabled"
+
+  Scenario: As a user, I can starts Minishift
```

Comment:
`As a user, I can start Minishift with registry-route addon enabled`

Reply:
@budhrg Do we really this ? i feel this is over explaining the scenario steps.

Reply:
@amitkrout I don't feel so, scenario should convey what are you doing in Given,When,Then.

If you see, you are actually verifying `Add-on 'registry-route' created docker-registry route.`